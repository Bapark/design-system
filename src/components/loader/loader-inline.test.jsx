import React from 'react';
import renderer from 'react-test-renderer';
import LoaderInline from './loader-inline.jsx';

describe('LoaderInline', () => {
  it('has defaults', () => {
    const tree = renderer.create((<LoaderInline />)).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
