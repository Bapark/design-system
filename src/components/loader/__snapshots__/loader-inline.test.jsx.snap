// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderInline has defaults 1`] = `
<div
  className="inlineWrapper"
  data-testid="loader"
>
  <span
    className="inline"
  >
    Loading...
  </span>
</div>
`;
